hbase-server/src/main/java/org/apache/hadoop/hbase/master/balancer/BalancerChore.java

```diff
@@ -46,7 +52,20 @@ public BalancerChore(HMaster master) {
   @Override
   protected void chore() {
     try {
-      master.balance();
+      BalanceResponse balanceResponse = master.balance();
```

Comment:
I wonder if it makes sense to move the below logic into master.balance(). I'm not saying for sure, I just wonder if we want it that we always update metrics whenever the balancer is invoked or only from the chore. This could be fine as is though

Reply:
Ya I think you are right. Keeping it in the chore sounds ok. Thanks